dockerfiles/platform-build-base/Dockerfile

```diff
@@ -67,6 +67,7 @@ RUN apt-get install -y \
   rsync \
   psmisc \
   libpopt0 \
+  zip \
```

Comment:
I think this is the wrong Dockerfile - you want this at runtime, not a build time.

Reply:
OK